tests/braynsTestData.cpp

```diff
@@ -213,6 +213,31 @@ BOOST_AUTO_TEST_CASE(render_circuit_with_particle_renderer)
                                  brayns.getEngine().getFrameBuffer()));
 }
 
+BOOST_AUTO_TEST_CASE(render_circuit_with_proximity_renderer)
+{
+    auto& testSuite = boost::unit_test::framework::master_test_suite();
+
+    const char* app = testSuite.argv[0];
+    const char* argv[] = {app,         "--renderer",
```

Comment:
The args are formatted weird. Why not:
```cpp
const char* argv[] = {app, "demo",
                      "--renderer", "proximity",
                      "--samples-per-pixel", "16",
                      "--synchronous-mode", "true"};
```

Reply:
The format applied here is what we agreed on when we decided to use clang-format. I would not see the point of making an exception here. And I don't see any issue with the new formatting, it's clear and not messy at all. Am I missing something?